src/features/dsComponents/meta/MetaEditor.tsx

```diff
@@ -234,9 +234,8 @@ const MetaEditor: React.FC<MetaEditorProps> = ({
               value={data[field.field]}
               placeholder={field.placeholder}
               maxLength={field.maxLength}
-              onChange={(e: React.FormEvent) => {
-                const target = e.target as HTMLInputElement
-                handleWrite(field.field, target.value)
+              onChange={(d: string) => {
+                handleWrite(field.field, d)
```

Comment:
this is a more general comment about the `DatasetEditor` & its children components (`TabbedDatasetEditor`, `ComponentEditor`, `MetaEditor` & `ReadmeEditor`), they don't seem to be used any longer. If they aren't being kept around for future use, we should remove them in this pr or a follow up!

Reply:
There may be bits in here we want to re-use, I'll make an issue for cleanup of the old editor components